server/item/writable_book.go

```diff
@@ -0,0 +1,100 @@
+package item
+
+type WritableBook struct {
+	Pages []string
+}
+
+// Page returns a specific page from the book. If the page exists, it will return the content and true, otherwise
+// it will return an empty string and false.
+func (w WritableBook) Page(page uint) (string, bool) {
```

Comment:
Is it neccessary to accept a uint? For API sake it will be better to accept a regular int and perform your own bound checks